<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KmlExportOptionWidget</class>
 <widget class="QWidget" name="KmlExportOptionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>104</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="colorStyleLabel">
     <property name="text">
      <string>Color style:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="colorStyleComboBox">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="startColorLabel">
     <property name="text">
      <string>Start color:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="allStartColorToolButton">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>All engines</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="jetStartColorToolButton">
       <property name="toolTip">
        <string>Jet engines</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="turbopropStartColorToolButton">
       <property name="toolTip">
        <string>Turboprop engines</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pistonStartColorToolButton">
       <property name="toolTip">
        <string>Piston engines</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="endColorLabel">
     <property name="text">
      <string>End color:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QToolButton" name="allEndColorToolButton">
       <property name="toolTip">
        <string>All engines</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="jetEndColorToolButton">
       <property name="toolTip">
        <string>Jet engines</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="turbopropEndColorToolButton">
       <property name="toolTip">
        <string>Turboprop engines</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pistonEndColorToolButton">
       <property name="toolTip">
        <string>Piston engines</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
